### ISSUE TITLE
[P3.2][Sourcing][Accessibility][Table , Multi-input , Date time picker][doc]Needs accessibility documentation

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10280]
#### Is this a bug, enhancement, or feature request?
Documentation for accessibility missing for all components in NGX website

#### Briefly describe your proposal.
We need documentation for core / platform components in NGX website.
mostly interested in complex components like : tree table , simple table, multi-input, datetimepicker etc

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Not applicable

#### If this is a bug, please provide steps for reproducing it.
We need documentation for core / platform components in NGX website

eg: 
 With the mouse you can adjust the column width. This is not possible with the keyboard
This was reported by customer because they don't know the shortcut keys  so we need help to add in NGX documentation about few accessibility section (in this case , its possible via SHIFT+ arrow left / right)

#### Is there anything else we should know?
Reported by customer

**Comment by droshev** [comment]
just documentation update, NO NEED TO DOWNPORT

**Comment by droshev** [comment]
agreed to be closed as it will be handled by sourcing team
